vgscollect/src/main/java/com/verygoodsecurity/vgscollect/core/VGSCollect.kt

```diff
@@ -675,40 +714,28 @@ class VGSCollect {
         fun setEnvironment(env: String): Builder = this.apply { environment = env }
 
         /** Sets the VGSCollect instance to use the custom hostname. */
```

Comment:
Sets the VGSCollect instance to use the custom hostname.
Also, the localhost IP can be used for VGS-Satellite for local testing.

@param cname where VGSCollect will send requests.